File: src/main/java/jp/artan/japanesefoodmod/init/tabs/SouvenirCreativeTab.java
package jp.artan.japanesefoodmod.init.tabs;

import jp.artan.japanesefoodmod.init.ItemInit;
import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;

public class SouvenirCreativeTab extends CreativeTabs {

    public SouvenirCreativeTab() {
        super("japanesefoodsouvenir");
    }

    @Override
    public ItemStack getTabIconItem() {
        return new ItemStack(ItemInit.TOKYO_BANANA);
    }

}

File: src/main/java/jp/artan/japanesefoodmod/init/tabs/SouvenirCreativeTab.kt
package jp.artan.japanesefoodmod.init.tabs

import jp.artan.japanesefoodmod.init.ItemInit
import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.ItemStack

class SouvenirCreativeTab : CreativeTabs("japanesefoodsouvenir") {

    override fun getTabIconItem(): ItemStack {
        return ItemStack(ItemInit.TOKYO_BANANA)
    }

}
